haskell/src/Lorentz/Contracts/Stablecoin.hs

```diff
@@ -114,7 +150,7 @@ instance Show Parameter where
   show = pretty
 
 instance ParameterHasEntrypoints Parameter where
-  type ParameterEntrypointsDerivation Parameter = EpdRecursive
+  type ParameterEntrypointsDerivation Parameter = EpdDelegate
```

Comment:
Why was it necessary to change `EpdRecursive` to `EpdDelegate`?

Reply:
Oh my bad, I forgot to undo that.